packages/adapters/src/adapters/_base/adapter.ts

```
@@ -0,0 +1,201 @@
+  extractEmojiVersion,
+  extractUnicodeVersion,
+  fetchCache,
+  MojisNotImplemented,
+} from "@mojis/internal-utils";
+import { defineMojiAdapter } from "../../adapter";
+
+function slugify(val: string): string {
+  return val.normalize("NFD")
+    .toLowerCase()
+    .trim()
+    .replace(/[\u0300-\u036F]/g, "")
```

Biome (1.9.4)

[error] A character class cannot match a character and a combining character.

A character and a combining character forms a new character. Replace the character class with an alternation.

(lint/suspicious/noMisleadingCharacterClass)